### DISCUSSION TITLE
Botão e menu de opções

### DISCUSSION THREAD

**Discussion by hugolimav8** [devlikeapro/waha#346] [Q&A]
Estou contribuindo e obtive o Plus, excelente projeto e documentação. Estou precisando de um menu e  botão de opções. Existe essa Api? Ou alguma outra forma? Diferente do Sendpoll, preciso de botões no menu.

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
Hi! Sorry, it's not available right now and unlikely we'll add it in the near future 
https://github.com/devlikeapro/whatsapp-http-api/issues?q=is%3Aissue+buttons+is%3Aclosed

Regular "send 1 to do something" always work :laughing: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
